src/pudl/package_data/eia923/column_maps/fgd_operation_maintenance.csv

```diff
@@ -6,8 +6,7 @@ opex_fgd_feed_materials_chemical_1000_dollars,feed_material_chemicals,feed_mater
 opex_fgd_labor_supervision_1000_dollars,,labor_and_supervision,labor_and_supervision,labor_and_supervision,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars
 opex_fgd_land_acquisition_1000_dollars,land_acquisition,,,,,,,,,,,,,,
 opex_fgd_waste_disposal_1000_dollars,waste_disposal,waste_disposal,waste_disposal,waste_disposal,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars
-oepx_fgd_maintenance_1000_dollars,,,,,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars
-opex_fgd_maintenance_material_other_1000_dollars,maintenance_material_other,maintenance_material_other,maintenance_material_other,maintenance_material_other,,,,,,,,,,,
+opex_fgd_maintenance_material_other_1000_dollars,maintenance_material_other,maintenance_material_other,maintenance_material_other,maintenance_material_other,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars
```

Comment:
What is the reasoning behind this change?

Reply:
Maintenance material and other is a stable category over time, that was getting mapped to two different columns depending on the original data source. It seems to be an error in the extraction, and I've collapsed this.